source/scene.js

```diff
@@ -146,7 +146,8 @@ function calcGridBounds( grid ) {
 this.executeBlock = function ( block, action ) {
     var blockName = block[ 0 ];
     var blockID = block[ 1 ];
-    this.blockExecuted( blockName, blockID );
+    //SJF: Was breaking procedures
+    // this.blockExecuted( blockName, blockID );
```

Comment:
Correct, this will break tracing for now. Possible next steps are to fix tracing, or to disable tracing for the dummy scenario / scenarios that use procedures.